src/components/sbb-link/sbb-link.tsx

```diff
@@ -67,25 +73,38 @@ export class SbbLink {
    */
   @Prop() public variant: InterfaceLinkAttributes['variant'] = 'block';
 
+  /**
+   * Disabled attribute if link is used as button
+   */
+  @Prop() public disabled?: boolean;
+
+  /**
+   * Name attribute if link is used as button
+   */
+  @Prop() public buttonName?: string;
+
+  /**
+   * Form attribute if link is used as button
+   */
+  @Prop() public formId?: string;
```

Comment:
We don't have a common pattern yet for internal ids. For form elements we agreed on `inputId`. Since this is not strictly speaking a form element, I would suggest to use `linkId` and use this id in all variants.
Also I would recommend initializing the id property. See e.g. https://github.com/lyne-design-system/lyne-components/blob/cc7e9a99253e64eaa3e6b479999b03d8f4fcc47f/src/components/sbb-title/sbb-title.tsx